<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_NOMBRE">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Nombre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_TEMP">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Temperatura</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_OXI">
        <property name="placeholderText">
         <string>Oxigenacion</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_PULSO">
        <property name="placeholderText">
         <string>Pulso</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_FECHA">
        <property name="placeholderText">
         <string>Fecha</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_GAS">
        <property name="placeholderText">
         <string>Gas</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_nu">
        <property name="text">
         <string>AGREGAR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_el">
        <property name="text">
         <string>ELIMINAR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_act">
        <property name="text">
         <string>ACTUALIZAR</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButton_find">
        <property name="text">
         <string>FIND</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_on">
        <property name="text">
         <string>ON</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_off">
        <property name="text">
         <string>OFF</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
